index.html

```diff
@@ -3816,6 +3836,56 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
 </pre>
           </aside>
         </section>
+        <!-- Maintain a fragment named "mASt-chunk" to preserve incoming links to it -->
+
+        <section id="mASt-chunk">
+          <h2><span class="chunk">mASt</span> mastering metadata</h2>
+
+          <p>The four decimal values below correspond to the four-byte mASt chunk type field:</p>
+
+          <pre>
+109 65 83 116
+</pre>
+          <p>If present, the <span class="chunk">mASt</span> chunk specifies the mastering display's metadata which includes
+          primaries, white point, maximum luminance, and minimum luminance as specified in [[SMPTE ST 2086]].</p>
+
+          <p>The following specifies the syntax of the <span class="chunk">mASt</span> chunk:</p>
+
+          <table id="mASt-chunk-syntax" class="numbered simple">
+            <caption>
+              mASt chunk components
+            </caption>
+
+            <tr>
+              <th>Name</th>
+              <th>Size</th>
+            </tr>
+
+            <tr>
+              <td>Mastering display colour primaries</td>
+              <td>12 bytes</td>
+            </tr>
+
+            <tr>
+              <td>Mastering display white point chromaticity</td>
+              <td>4 bytes</td>
+            </tr>
+
+            <tr>
+              <td>Mastering display maximum luminance</td>
+              <td>4 bytes</td>
+            </tr>
+          </table>
```

Comment:
I would provide at least one example, e.g.:

![image](https://user-images.githubusercontent.com/4871350/205381821-4e08daf8-a940-4c8e-83ce-525c2323b308.png)

Reply:
Agreed @ providing examples.
I was a little hesitant because for the example to make sense, I would need to specify the units. And once I do that, I'm basically duplicating the entire spec I'm referencing.

Reply:
Done.